ruby/constants/menuOptions.rb

```diff
@@ -0,0 +1,14 @@
+@menuOptions = {
+  :'1' => {path: './ruby/exercises/guess-the-number.rb', name: 'Guess The Number'},
```

Comment:
Not sure which Ruby version you were targeting - newer version of Ruby (2 and above, iirc) allow us to use colon syntax for key/value declarations:
```ruby
{
  1: { path: 'etc', name: 'etc' }
}
```